src/node_v8.cc

```diff
@@ -60,6 +60,13 @@ void GetHeapStatistics(const FunctionCallbackInfo<Value>& args) {
 
 
 void SetFlagsFromString(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (args.Length() < 1)
+    return env->ThrowTypeError("v8 flag is required");
```

Comment:
Yes, that might be right, but `args.Length()` is an integer value that should be between 1 and infinity.